### ISSUE TITLE
OTP 21+ support?

### ISSUE THREAD

**Issue by JuliusFox** [basho/riak-erlang-client#380]
When to support new versions?

**Comment by aboroska** [comment]
It does work with OTP21 and OTP22. You need to use a recent `rebar3` to compile though. Ignore the checked-in rebar which is obsolete now.

**Comment by daniel020403** [comment]
Yes, I did try compiling it and it works fine with OPT22. However, there are already more recent versions and it doesn't compile successfully on my end with those. It was OTP24 on my end.

**Comment by martinsumner** [comment]
Release of OTP 24 support is pending, hope is to complete the work in the next 4-6 weeks to make Riak 3.2.

The WIP for the erlang client is in this branch - https://github.com/basho/riak-erlang-client/tree/develop-3.2-otp24.  This should still work against older versions of Riak.
